.github/workflows/go_mod_tidy.yml

```
@@ -3,30 +3,17 @@ on:
   pull_request:
     types:
       - labeled
-  workflow_dispatch:
-    inputs:
-      pr_number:
-        description: "PR number"
-        required: true
-        type: number
 
 jobs:
   mod_tidy_and_generate_licenses:
-    if: ${{ github.event_name == 'workflow_dispatch' || (github.event_name == 'pull_request' && github.actor == 'dependabot[bot]' && contains(github.event.pull_request.labels.*.name, 'dependencies-go')) }}
+    if: ${{ github.event_name == 'pull_request' && github.actor == 'dependabot[bot]' && contains(github.event.pull_request.labels.*.name, 'dependencies-go') }}
```

```suggestion
    if: ${{ github.event.pull_request.user.login == 'dependabot[bot]' && contains(github.event.pull_request.labels.*.name, 'dependencies-go') }}
```
The event can only be a PR now, and we were told using the `github.actor` is less secure than the user of the event